invenio_rdm_migrator/streams/runner.py

```diff
@@ -91,7 +97,6 @@ def __init__(self, stream_definitions, config_filepath):
                             existing_data=existing_data,
                             **stream_config.get("load", {}),
                         ),
-                        logger=self.logger,
```

Comment:
What I found online they avoid drilling the attribute, using the getter instead. In some cases they create factories (i.e. encapsulate the getter, even cache it). I think for now, that is a bit over-engineering it.